rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/FileMetadata.java

```diff
@@ -382,4 +395,19 @@ public int getGenerationNumber()
   {
     return generationNumber;
   }
+
+  static class Pair {
+    int generationNumber;
```

Comment:
scope can be private here.

Reply:
+1, add scope for both vars. And please follow the coding convention of file for newlines, spacing and bracket placements.